tests/unit_tests/utils/test_general.py

```diff
@@ -11,7 +11,11 @@
 from astropy.io.misc import yaml
 
 import simtools.util.general as gen
-from simtools.util.general import InvalidConfigEntry, UnableToIdentifyConfigEntry
+from simtools.util.general import (
+    InvalidConfigEntry,
```

Comment:
Not sure I understand this comment. These are not configuration modules but just exceptions we created to make the errors a bit more informative.

Reply:
What I meant is that we are using `general.py` for some "specific" purposes. For example, the use of `validate_config_data` is specific to `self.config`. I also think we have to be careful when using the word "config" or "configuration" because at this point we already have lots of different configurations. Anyways, nothing to do here, hence I am closing the conversation.